Food/zig.lua

```diff
@@ -0,0 +1,50 @@
+local name = "zig"
+local version = "0.9.0"
+
+food = {
+    name = name,
+    description = "Zig is a general-purpose programming language and toolchain for maintaining robust, optimal, and reusable software.",
+    license = "MIT (Expat)",
```

Comment:
You can shorten this to “MIT” - IIRC the expat variant is the one that is most commonly referred to as the MIT license. The X11 variant is the one with additional anti-advertising clauses.

Reply:
Ah great thanks for explaining and for the info/review. I have committed the change now.